implementations/rust/ockam/ockam_command/src/node/show.rs

```diff
@@ -180,77 +63,51 @@ pub async fn print_query_status(
     is_default: bool,
 ) -> miette::Result<()> {
     let cli_state = opts.state.clone();
-    if !is_node_up(ctx, node_name, node, cli_state.clone(), wait_until_ready).await? {
-        let node_state = cli_state.nodes.get(node_name)?;
-        let node_port = node_state
-            .config()
-            .setup()
-            .api_transport()
-            .ok()
-            .map(|listener| listener.addr.port());
-
-        // it is expected to not be able to open an arbitrary TCP connection on an authority node
-        // so in that case we display an UP status
-        let is_authority_node = node_state.config().setup().authority_node.unwrap_or(false);
-        print_node_info(
-            opts,
-            node_port,
-            node_name,
-            is_default,
-            is_authority_node,
-            None,
-            None,
-            None,
-            None,
-            None,
-        );
-    } else {
-        let node_state = cli_state.nodes.get(node_name)?;
-        // Get short id for the node
-        let default_id = match node_state.config().identity_config() {
-            Ok(resp) => resp.identifier().to_string(),
-            Err(_) => String::from("None"),
-        };
+    let node_state = cli_state.nodes.get(node_name)?;
+    let node_port = node_state
```

Comment:
Note that the port won't be available until the node is up and running. Also note that the `is_node_up` function is looping a few times to wait unti the node is up, that's why this function must be the first to run before querying any node information. Same problem with the `node_state`: if the node is not up when you call it, you won't be getting all the node information.

Reply:
Reverted to previous way of doing it